src/bindings/js/node/src/helper.cpp

```diff
@@ -106,14 +77,14 @@ std::vector<size_t> js_to_cpp<std::vector<size_t>>(const Napi::CallbackInfo& inf
         auto data_ptr = static_cast<int*>(buf.ArrayBuffer().Data());
         std::vector<size_t> vector(data_ptr, data_ptr + buf.ElementLength());
         return vector;
+    } else {
+        OPENVINO_THROW("Passed argument must be of type Array or TypedArray.");
     }
```

Comment:
If else is last instruction in the function body we can white it out of condition
```suggestion
    }
    OPENVINO_THROW("Passed argument must be of type Array or TypedArray.");
```

Reply:
I would like to keep it for the sake of readability